src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ProductReview.xml

```diff
@@ -20,31 +20,38 @@
             <group>product_review:read</group>
         </attribute>
         <attribute name="createdAt">
-           <group>product_review:read</group>
-       </attribute>
+            <group>product_review:read</group>
```

Comment:
As you've touched this place anyway, can we adjust previous serialization groups to `admin:product_review:read`? Can be done in separate PR